### PR TITLE
Added velocity to swiping event callback

### PR DESCRIPTION
**Pull request by jakedowns**
Dunno if anyone else will find this useful, but I found myself needing a reference to the current velocity for my page swipe indicator.

Might be better to just expose velocity on props object...

(Sorry for the dupe commit messages, was trying to rebase/squash but it didn't do what I expected.)

### CODE REVIEW

File: jsx/Swipeable.jsx [hunk 1]
```diff
@@ -75,7 +79,7 @@ var Swipeable = React.createClass({
     }
 
     if (this.props.onSwiping) {
-      this.props.onSwiping(e, pos.deltaX, pos.deltaY, pos.absX, pos.absY)
+      this.props.onSwiping(e, pos.deltaX, pos.deltaY, pos.velocity)
```
Comment (goatslacker) [inline jsx/Swipeable.jsx]: why not also send the x and y?

Comment (jakedowns) [inline jsx/Swipeable.jsx]: good point, re-added

**Comment by goatslacker** [comment]
Can you rebase this down to a single commit?

**Comment by hartzis** [comment]
@goatslacker I like this addition but I don't know if @jakedowns is still around to rebase. We can just add these changes in the next couple of updates with the build changes.

**Comment by jakedowns** [comment]
Closing (superseded by #33 )

**Pull request closed without merging**